### PR TITLE
Fixes xenomorph hiss emote sounds because I am stupid.

### PR DESCRIPTION
**Pull request by Razoozle**
Basically this just fixes it so it cycles between different hisses rather than just one. I didn't know if it would work or not.

### CODE REVIEW

File: code/modules/mob/living/carbon/alien/humanoid/emote.dm [hunk 1]
```diff
@@ -76,7 +76,7 @@
 			playsound(src.loc, 'sound/voice/hiss5.ogg', 40, 1, 1)
 
 		if (act == "hiss")
-			playsound(src.loc, 'sound/voice/hiss3.ogg', 40, 1, 1)
+			playsound(src.loc, "hiss", 40, 1, 1)
```
Comment (tkdrg) [inline code/modules/mob/living/carbon/alien/humanoid/emote.dm]: shouldnt this be `get_sfx("hiss")`

Comment (Ultimate-Chimera) [inline code/modules/mob/living/carbon/alien/humanoid/emote.dm]: Not sure, but I tested this and it worked.

Comment (Ultimate-Chimera) [inline code/modules/mob/living/carbon/alien/humanoid/emote.dm]: Nope, get_sfx("hiss") doesn't seem to work.

Comment (KazeEspada) [inline code/modules/mob/living/carbon/alien/humanoid/emote.dm]: playsound() does the get_sfx()

Comment (Ultimate-Chimera) [inline code/modules/mob/living/carbon/alien/humanoid/emote.dm]: I really don't know byond code, but this works fine. What's different about get_sfx()?

**Comment by tkdrg** [comment]
I read sound code wrong nevermind me

**Pull request merged**